### PR TITLE
🚩 Milestone 4: OOP school library | feature/addBasicUI

### PR DESCRIPTION
**Pull request by Ruthmy**
### Hi there! Ruthmy here. 👋

This is the PR for the [`OOP school library: add basic UI`](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md) activity. The goal of this milestone is to create a basic UI for the library program. The UI should allow the user to:
- [x] List all books.
- [x] List all people.
- [x] Create a person (teacher or student).
- [x] Create a book.
- [x] Create a rental.
- [x] List all rentals for a given person id.
- [x] Lets users choose an option.
- [x] If needed, ask for parameters for the option.
- [x] Stop the program.

### Here is a summary of what has been done:

- [x] Create the `app.rb` file to hold the main program and the logic for the UI.
- [x] Create the `main.rb` file to execute the program.

---

### Thank you for taking the time to review this PR ⭐!

---

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    if book.nil? || person.nil?
+      puts "\nBook or person not found, please try again.\n\n"
+      sleep 3
+      create_rental
+    else
+      rental = Rental.new(date, book, person)
+      @rentals.push(rental)
+      puts "\nRental created successfully"
+      sleep 3
+      run
```
Comment (usorfaitheloho) [inline app.rb]: You did well with your project, kindly note that according to the app implementation given, when creating a `rental`, a person should be selected by `number` and not `id` as seen in your own application. Please implement this change to follow the requirement for the project 😄 

Video screenshot :
image

Your app :
image

This is the link of the [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Ruthmy  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> *Interactive console app functions properly✔️

> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (Ruthmy) [reply app.rb]: Hi @usorfaitheloho 
Thank you for your feedback! Yes, you are right, at that point you should enter the number. In my application also works by entering the number, but by mistake I put ID in the text to display. I will fix it right away and upload the changes.
Thank you very much! ✨

**Review by prolajumokeoni — APPROVED** [review]
Hi @Ruthmy 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - Updated app to use `number` not `id` when creating a rental ✅ 
   

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**